default.nix

```diff
@@ -1,20 +1,56 @@
 { system ? builtins.currentSystem
 , crossSystem ? null
 , config ? {}
-
 # Import IOHK common nix lib
-, iohkLib ? import ./nix/iohk-common.nix
-# Pin nixpkgs to a revision on the nixos-19.03 channel
-, nixpkgs ? iohkLib.fetchNixpkgs ./nix/nixpkgs-src.json
-, pkgs ? import nixpkgs { inherit system crossSystem config; }
+, iohkLib ? import ./nix/iohk-common.nix { inherit system crossSystem config; }
+# Use nixpkgs pin from iohkLib
+, pkgs ? iohkLib.pkgs
 
 # Keep this argument even if unused.
 # It will prevent Hydra from caching the evaluation.
 , gitrev ? iohkLib.commitIdFromGitRepo ./.
```

Comment:
this will only trigger a new eval if its actually an input to a job

it needs to be routed to the forceNewEval job that iohk-nix is generating

Reply:
Good point. I have removed it from here, and made it local to the `required` aggregate job.